spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe "the Ingredients index page" do
+  it "should show a list of all the ingredients including their name and cost" do
+     ingredient1 = Ingredient.create(name: "Apple", cost: 1)
+     ingredient2 = Ingredient.create(name: "Pie Crust", cost: 2)
+     ingredient3 = Ingredient.create(name: "Pie Filling", cost: 2)
+     visit "/ingredients"
+
+      expect(page).to have_content("Apple: 1")
```

Comment:
Let's get your tests a little more robust. We can use `within` blocks to help specify what/where things show up on our page. 
```ruby
within "#ingredients" do
   expect(page).to have_content("#{ingredient1.name}: #{ingredient1.cost}")        
   expect(page).to have_content("#{ingredient2.name}: #{ingredient2.cost}")        
   expect(page).to have_content("#{ingredient3.name}: #{ingredient3.cost}")
end
```